packages/web-sdk/src/transports/otlp/proposal-logs-payload-1.json

```diff
@@ -0,0 +1,465 @@
+          {
+            "key": "telemetry.sdk.version", // OTEL SEMANTIC CONVENTIONS
+            "value": { "stringValue": "1.0.0-beta2" }
+          },
+
+          {
+            "key": "session.id", // FARO
+            "value": { "stringValue": "abcdefg" }
+          },
+          {
+            "key": "session.attributes", // FARO
+            "value": { "kvlistValue": [] }
+          },
+
+          // Taken from GENERAL IDENDITY ATTRIBUTES. Some may differ from Faro, some only exist in Faro.
+          // https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/span-general.md#general-identity-attributes
+          {
+            "key": "enduser.id", // OTEL SEMANTIC CONVENTIONS
+            "value": { "stringValue": "119" }
+          },
+          {
+            "key": "enduser.name", // FARO
+            "value": { "stringValue": "119" }
+          },
+          {
+            "key": "enduser.email", // FARO
+            "value": { "stringValue": "119" }
+          },
+          {
+            "key": "enduser.attributes", // FARO
+            "value": { "kvlistValue": [] }
+          },
+
+          {
+            "key": "app.name", // FARO
+            "value": { "stringValue": "foo" }
+          },
+          {
+            "key": "app.release", // FARO
+            "value": { "stringValue": "6.6.7." }
+          },
+          {
+            "key": "app.version", // FARO
+            "value": { "stringValue": "1.23" }
+          },
+          {
+            "key": "app.environment", // FARO
+            "value": { "stringValue": "production" }
+          }
+        ],
+
+        // We'll hardcode this to 0
+        "droppedAttributesCount": 0
+      },
+
+      "scopeLogs": [
+        {
+          // The scope should originate from the instrumentation that captured this
+          // For the time being we don't have this information
+          // We will use the sdk as the source of information
+          "scope": {
+            "name": "@grafana/faro-core",
+            "version": "1.0.0-beta4"
+          },
+          "logRecords": [
+            // +++ EVENT +++
+            {
+              // We convert the timestamp to milliseconds and then multiply them by 1 million
+              // The timestamp comes from the timestamp property on the payload
+              "timeUnixNano": 1674464646296000000,
+              // We'll use the same value from timestamp property on the payload
+              "observedTimeUnixNano": 1674464646296000000,
+              // We can just use 1 and TRACE as values for event signals
+              "severityNumber": 1,
+              "severityText": "TRACE",
+              "body": {
+                "stringValue": "click_button_with_attributes"
+              },
+              "attributes": [
+                // This should be a convention to identify events from the other logs
+                {
+                  "key": "faro.event", // FARO
```

Comment:
IMHO unnecessary to specify that event is an event :-) exceptions/measurements can be extracted by `event.name`